html/table.html

```diff
@@ -448,8 +449,8 @@
                 const svg = d3.create("svg")
                     .attr("id", "TreeSVG")
                     .attr("width", width + 20) //makes sure all tree nodes are within view
-                    .attr("style", `width:100%; height: auto; font: 10px sans-serif; user-select: none;`)
-                    
+                    .attr("style", `width:100%; height: auto; font: 10px sans-serif; user-select: auto;`) //user-select:auto makes text selectable
```

Comment:
Thank you for updating this! This will have to be updated in the code for each tree however. I can assist with that if needed! just let me know.

Reply:
fixed in: https://github.com/phac-nml/ArborView/pull/8/commits/e1eb377672aa45e09c1fcd5c6335c6006417e864